biocrnpyler/mechanism.py

```diff
@@ -605,6 +605,136 @@ def update_reactions(self, dna, component = None, kexpress = None,
             protein = Species(dna.name, material_type="protein")
         rxns = [Reaction(inputs=[dna], outputs=[dna, protein], k = kexpress)]
         return rxns
+    
+class multi_tx(Mechanism):
```

Comment:
Hi Matthieu,
Would you mind adding at least a one-line comment to this and other classes in your PR, so users can know how to use/what to expect from this class? Many thanks!

Reply:
Hey Zoltuz,
Should I just add the description to the doc-string of the class or somewhere else? I can also refer them to the demo notebook in this PR.

Reply:
docstring of a class would be the best place. Thanks!

Reply:
Just updated with docstring and directing to demo nb for detailed mechanism and demonstration.